packages/expo-ads-admob/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Provides support for the Google AdMob SDK (https://www.google.com/admob/) for mobile advertising. This module is largely based of the react-native-admob (https://github.com/sbugert/rea ct-native-admob) module, as the documentation and questions surrounding that module may prove helpful. A simple example implementing AdMob SDK can be found at https://github.com/deadcoder0904/expo-goo gle-admob",
```

Comment:
@EvanBacon some links in the `description` fields are broken & also on README

Reply:
can you elaborate?

Reply:
@brentvatne the links in the description have empty spaces in between. for eg: `(https://github.com/sbugert/rea ct-native-admob)` & `https://github.com/deadcoder0904/expo-goo gle-admob`. i was just checking `expo-google-admob` docs & came across this :) they were also messed up in its README

Reply:
fixed, thanks!